scripts/populate_panels/load_g2p_data.pl

```diff
@@ -376,6 +408,9 @@ sub add_annotations {
   $count = add_comments($gfd, $comments, $user);
   print $fh_report "    Added $count comments\n" if ($count > 0);
 
+  $count = add_public_comments($gfd, $public_comments, $user);
+  print $fh_report "    Added $count comments\n" if ($count > 0);
```

Comment:
It should be related to public comments:
print $fh_report "    Added $count public comments\n" if ($count > 0);